docs/content/en/docs/packages/packagecontroller.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Manging the package controller"
+linkTitle: "Package controller"
+weight: 4
+aliases:
+    /docs/tasks/packages/packagecontroller/
+---
+
+### Install package controller after installation
```

Comment:
this is kinda confusing. Do you mean "Reinstall package controller"?

Reply:
I moved this from the prequisite page, so that is correct. However, it is the same process for both. I can probably clarify this.

I think I can just name this section "Installing the package controller" and then add context below as to when this would be used.